snapshots/climate/2024-02-22/weekly_wildfires_2003_2023.py

```diff
@@ -0,0 +1,408 @@
+"""Script to create a snapshot of dataset. Loads data from the EFFIS API and creates a snapshot of the dataset with weekly wildire numbers, area burnt and emissions.
+This script generates data from 2003-2023. The data for the year 2024 and above will be processed separately to avoid long processing times."""
+
+from pathlib import Path
+
+import click
+import pandas as pd
+import requests
+from owid.datautils.io import df_to_file
+from structlog import get_logger
+from tqdm import tqdm
+
+from etl.snapshot import Snapshot
+
+# Initialize logger.
+log = get_logger()
+
+
+# Version for current snapshot dataset.
+SNAPSHOT_VERSION = Path(__file__).parent.name
+# Define a range of years for which data will be processed.
+YEARS = range(2003, 2024)
+
+COUNTRIES = {
```

Comment:
Nit - you could load countries from regions dataset like this
```
from owid.catalog import find
tb_regions = find(table="regions", dataset="regions").iloc[0].load()
```
(it's not by any means necessary, feel free to ignore it)

Reply:
thanks! that's actually a useful function to know